docs/stdlib/numbers.rst

```diff
@@ -504,12 +523,12 @@ from :eql:type:`str` and :eql:type:`json`.
                   std::round(value: decimal) -> decimal
                   std::round(value: decimal, d: int64) -> decimal
 
-    Round to the nearest value.
+    Returns a number from ``value`` rounded by its nearest integer.
```

Comment:
```suggestion
    Returns the given number rounded to its nearest value.
```

Reworded since rounding doesn't always result in an integer. (See the last signature.)